src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -217,7 +217,7 @@ def __call__(self, *args, **kwds) -> Any:
         return self.get_state()
 
 
-class PyReadOnlyStateContainer(PyCallableStateObject):
+class PyStateContainer(PyCallableStateObject):
     """Object class using StateEngine based DatamodelService as backend. Use
     this class instead of directly calling DatamodelService's method.
 
```

Comment:
Can we list `__call__` as a method? Then we don't need to tag it on to the end of get_state / set_state descriptions, which I find unclear.

Reply:
@seanpearsonuk, thank you for the clarification. This has been implemented.